src/main/java/Avalon.java

```diff
@@ -0,0 +1,136 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
```

Comment:
Perhaps write a header comment for the class to improve code readability? I noticed that this is missing in other classes/public methods that were added in this PR as well.